srcjs/shinyapp.js

```diff
@@ -632,7 +632,12 @@ var ShinyApp = function() {
 
   addMessageHandler('config', function(message) {
     this.config = {workerId: message.workerId, sessionId: message.sessionId};
+    // != is on purpose (instead of !==) because it checks for both null and undefined
+    // (see here: http://stackoverflow.com/a/21273362/6174455)
+    /*eslint-disable */
     if (message.user != null) exports.user = message.user;
```

Comment:
It's more idiomatic to use `if (message.user)` (to look for truthy values in general) or, if you want to be more precise, you could be explicit with `(message.user !== undefined && message.user !== null)`. Either way is cleaner than adding the (necessary) comments explaining why it's written that way.